lib/dgram.js

```diff
@@ -134,7 +135,7 @@ function replaceHandle(self, newHandle) {
   self._handle = newHandle;
 }
 
-Socket.prototype.bind = function(port_ /*, address, callback*/) {
+Socket.prototype.bind = promisify(function(port_ /*, address, callback*/) {
```

Comment:
quick clarification question: bind is one of those methods that can throw in some cases, even with a callback. that is, `If binding fails, an 'error' event is generated. In rare case (e.g. attempting to bind with a closed socket), an Error may be thrown.` ... How is that handled with this approach? Is the Error still thrown or is it encapsulated into the Promise logic if the callback is not provided? Since the `callback` is already optional, existing code that does not currently pass a callback and is not paying attention to the return value could end up losing important errors if the throws get wrapped into the Promise.